website/app/global.css

```
@@ -1,112 +1,87 @@
+  --color-fd-input: #333333;
+  --color-fd-ring: #ffffff;
+  --canonfig-accent: #ffffff;
+  --canonfig-surface: #000000;
+  
+  /* Reset animations */
+  --animation-duration: 0ms !important;
+  --transition-duration: 0ms !important;
 }
 
 body {
   text-rendering: optimizeLegibility;
```

Stylelint (17.14.0)

[error] Expected "optimizeLegibility" to be "optimizelegibility" (value-keyword-case)

(value-keyword-case)